canvas_modules/harness/assets/images/down-triangle.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024" height="10" width="10"><path d="M42 128l469 810 469-810z"/></svg>
+<svg width="10" height="5" fill-rule="evenodd"><path d="M0 0l5 4.998L10 0z"></path></svg>
```

Comment:
Why is the height and width set to 10 and 5?

When I downloaded the caret image from [Carbon library](https://carbondesignsystem.com/guidelines/icons/library/), this is the content:

```
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 24.0.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="32px" height="32px" viewBox="0 0 32 32" style="enable-background:new 0 0 32 32;" xml:space="preserve">
<style type="text/css">
	.st0{fill:none;}
</style>
<title>caret--down</title>
<polygon points="24,12 16,22 8,12 "/>
<rect id="_Transparent_Rectangle_" class="st0" width="32" height="32"/>
</svg>

```

how did you end up with that file?

Reply:
Hi @caritaou , I did grab them from the test harness,  are these files relevant?
<img width="588" alt="image" src="https://user-images.githubusercontent.com/89609176/206523837-7596f95b-0e2f-4228-857c-5b39051fad9d.png">
<img width="588" alt="image" src="https://user-images.githubusercontent.com/89609176/206523879-f7ef912e-dfdc-4b84-9299-3e5375b21290.png">

Reply:
@nabilIbm You have to download carbon icons SVGs from here - https://v10.carbondesignsystem.com/guidelines/icons/library
<img width="946" alt="Screen Shot 2022-12-08 at 10 02 47 AM" src="https://user-images.githubusercontent.com/25124000/206530176-1dc2ab75-fc2d-478d-b3e0-a5bd503dff65.png">

Reply:
@tomlyn Unfortunately that approach will not work for the Toggletext control. We allow applications to pass in custom images to display https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/controls/toggletext/toggletext.jsx#L53.

This issue is to update the svg files used in the examples in the test harness to come from Carbon and to set generic styles in the Toggletext control to align the icons

Reply:
Oh OK, I see, so you don't need Carbon icons for this then you can just use any SVG file we have.